code/modules/power/singularity/narsie.dm

```diff
@@ -29,8 +29,11 @@
 	. = ..()
 	icon_state = SSticker.cultdat?.entity_icon_state
 	name = SSticker.cultdat?.entity_name
-	to_chat(world, "<font size='15' color='red'><b> [uppertext(name)] HAS RISEN</b></font>")
-	SEND_SOUND(world, sound(pick('sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg')))
+
+	for(var/mob/living/player in GLOB.player_list)
+		if(!isnewplayer(player))
+			to_chat(player, "<font size='15' color='red'><b> [uppertext(name)] HAS RISEN</b></font>")
+			SEND_SOUND(player, sound(pick('sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg')))
```

Comment:
This isn't perfomant.  

Cache the sound locally, first, please.

Reply:
Would this be a better way to do it?

```dm
var/list/sound = list('sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg')

for (var/mob/living/player in GLOB.player_list)
  if (!isnewplayer(player))
    to_chat(player, "<font size='15' color='red'><b> [uppertext(name)] HAS RISEN</b></font>")
    SEND_SOUND(player, sound(pick(sound)))
```

Or drop the `pick()` function and just

```dm
var/sound = pick('sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg')

for (var/mob/living/player in GLOB.player_list)
  if (!isnewplayer(player))
    to_chat(player, "<font size='15' color='red'><b> [uppertext(name)] HAS RISEN</b></font>")
    SEND_SOUND(player, sound)
```

I'm leaning more to the first code snippet as this way you could get both sound effects.

Reply:
Forgot to mention @Fox-McCloud

Reply:
```
	var/im_here = sound(pick('sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg'))
	
	for(var/mob/living/L in GLOB.player_list)
		if(isnewplayer(L))
			continue
		to_chat(L, "<font size='15' color='red'><b> [uppertext(name)] HAS RISEN</b></font>")
		SEND_SOUND(L, im_here)
```